packages/amazonq/src/app/chat/activation.ts

```diff
@@ -57,7 +57,10 @@ export async function activate(context: ExtensionContext) {
 
     await amazonq.activateBadge()
     void setupLsp()
-    void setupAuthNotification()
+
+    // Display notification to sign if the user hasn't yet.
+    // Abitrary wait time to give the auth restore routine time to initialize.
+    globals.clock.setTimeout(() => void setupAuthNotification(), 5000)
```

Comment:
AuthUtil emits an onDidChangeActiveConnection event, is it possible we can wait for that to trigger and then we call setupAuthNotification?

Reply:
Does it emit if there is no connection? That's when we would display the notification

Reply:
Nvm with my earlier comments, I guess since this notification isn't critical we can prob just keep it simple.

Reply:
replied to Justin